skimage/restoration/tests/test_restoration.py

```diff
@@ -2,7 +2,9 @@
 import pytest
 from scipy.signal import convolve2d
 from scipy import ndimage as ndi
+from skimage._shared import testing
```

Comment:
It's a detail, but I think that we must use `@pytest.mark.parametrize` instead of its alias `@testing.parametrize`

Reply:
yes, good catch